packages/wallet/src/redux/selectors.ts

```diff
@@ -67,3 +67,24 @@ export const getProtocolForProcessId = (
 export const getProtocolState = (state: walletStates.Initialized, processId: string) => {
   return state.processStore[processId].protocolState;
 };
+
+export const getLatestNonce = (
+  state: SharedData,
+  participantA: string,
+  participantB: string,
+  libraryAddress: string,
```

Comment:
What about just using the latest nonce for the given `libraryAddress`?

Reply:
We can do that. It just means we burn through nonces faster.